src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StaffSectionDim/v_2/11324_XLTV31_should_have_ElectronicMailAddress_Empty.xml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<TestCase>
+  <DBMS>Any</DBMS>
+  <ControlDataInsertion>
+  </ControlDataInsertion>
+  <Query>
+      SELECT ElectronicMailAddress
+      FROM analytics.StaffSectionDim
+      WHERE StaffSectionKey = '11324-867530010-A01-702-XLTV31-540-2012-14139-1';
+  </Query>
+  <Result>
+      <ElectronicMailAddress></ElectronicMailAddress>
```

Comment:
We test for ElectronicMailAddress and HighestedCompletedLevelOfEducation being empty since they can be nullable, but we do not test for any of the other nullable fields (such as YearsOfPriorProfessionalExperience and YearsOfTeachingExperience)  being empty. Should there be tests for those as well?

Reply:
I am replacing null value with zero. I renamed three testcases to show that. For 13724_BPR20 I changed 13724_BPPR20_should_have_HighlyQualifiedTeacher_Zero, 13724_BPPR20_should_have_YearsOfPriorProfessionalExperience_Zero, 13724_BPPR20_should_have_YearsOfPriorTeachingExperience_Zero. Those testcases already validated null, but the view replace null with a 0. I didn't change any testcases, only the names.